.github/workflows/tests.yml

```diff
@@ -0,0 +1,86 @@
+name: "Tests: PR"
+
+on:
+  push:
+    branches:
+      - main
+      - develop
+      - ci-gha
```

Comment:
Branch names will need to be adjusted as we go. `ci-gha` was only added to run this branch on the CI of the fork.